config/deploy.rb

```diff
@@ -25,10 +25,10 @@
 # set :pty, true
 
 # Default value for :linked_files is []
-set :linked_files, %w{config/database.yml config/solr.yml config/default_htaccess_directives}
+set :linked_files, %w{config/database.yml config/solr.yml config/settings/staging.yml}
 
 # Default value for linked_dirs is []
-set :linked_dirs, %w{log config/certs config/environments tmp/pids tmp/cache tmp/sockets vendor/bundle public/system}
+set :linked_dirs, %w{log config/certs tmp/pids tmp/cache tmp/sockets vendor/bundle public/system}
```

Comment:
good point.  could then get rid of `config/settings/staging.yml` from `:linked_files`.